src/iOS/toga_iOS/widgets/switch.py

```diff
@@ -0,0 +1,44 @@
+from rubicon.objc import objc_method
+
+from toga.interface import Switch as SwitchInterface
+
+from .base import WidgetMixin
+from ..libs import *
+
+
+class TogaSwitch(UISwitch):
+    @objc_method
+    def onPress_(self, obj) -> None:
+        if self._interface.on_press:
+            self._interface.on_press(self._interface)
+
+
+class Switch(SwitchInterface, WidgetMixin):
+    def __init__(self, label, id=None, on_press=None, style=None, state=False):
+        super().__init__(label, id=id, style=style, on_press=on_press, state=state)
+        self._create()
+
+    def create(self):
+        self._impl = TogaSwitch.alloc().init()
+        self._impl._interface = self
+
+        self._impl.addTarget_action_forControlEvents_(self._impl, get_selector('onPress:'), UIControlEventValueChanged)
```

Comment:
I get why you haven't put a label here, but the inconsistency concerns me. How would you envisage using "Switch" in a real-world cross-platform situation?